README.md

```diff
@@ -1,14 +1,38 @@
-## Overview
-[![Build Status](https://travis-ci.org/openebs/velero-plugin.svg?branch=master)](https://travis-ci.org/openebs/velero-plugin)
-[![Go Report](https://goreportcard.com/badge/github.com/openebs/velero-plugin)](https://goreportcard.com/report/github.com/openebs/velero-plugin)
+# Velero-plugin for OpenEBS CStor volume
 
 Velero is a utility to back up and restore your Kubernetes resource and persistent volumes.
 
 To do backup/restore of OpenEBS CStor volumes through Velero utility, you need to install and configure
 OpenEBS velero-plugin.
 
+[![Build Status](https://travis-ci.org/openebs/velero-plugin.svg?branch=master)](https://travis-ci.org/openebs/velero-plugin)
+[![Go Report](https://goreportcard.com/badge/github.com/openebs/velero-plugin)](https://goreportcard.com/report/github.com/openebs/velero-plugin)
+
+## Table of Contents
+- [Compatibility matrix](#compatibility-matrix)
+- [Prerequisite for velero-plugin](#prerequisite-for-velero-plugin)
+- [installation of velero-plugin](#installation-of-velero-plugin)
+- [Configuring snapshot location](#configuring-snapshot-location)
+- [Managing Backups](#managing-backups)
+  - [Creating a backup](#creating-a-backup)
+  - [Creating a restore from backup](#creating-a-restore-from-backup)
+  - [Creating a scheduled backup](#creating-a-scheduled-backup-or-incremental-backup-for-cstor-volume)
+  - [Restoring from a scheduled backup](#restoring-from-a-scheduled-backup)
+
+## Compatibility matrix
+
+|     Image                |    Codebase     |  Velero v0.10.0  | Velero v0.11.0 | Velero v1.0.0 | OpenEBS/Maya release |
+|   -------------------    |  ---------------|   ---------      |  --------------- |  -----------------  | -------- |
```

Comment:
Just thinking aloud here. The user might start with a given OpenEBS version and then Velero version and ask for what is the correct Velero Pluggin to install. 

- So should the table be of the format:
  column 1 - openebs/maya version
  column 2 - velero version 
  column 3 - velero plugin image
  column 4 - code base.

Reply:
yes. That make sense.